src/platform/linux/misc.cpp

```diff
@@ -8,13 +8,15 @@
 
 #include <fstream>
 
+// local includes
```

Comment:
Not sure if you picked up on this from some of my other code, but thanks for adding these. `// standard includes` at L1 too?

Normally I put "lib includes" before "local" unless there's some conflict with it.

Reply:
I'll arrange them to match the others. I was just nudging it towards the standard not to mix another change in the PR.